docs/hazmat/primitives/mac/poly1305.rst

```diff
@@ -85,3 +85,44 @@ messages allows an attacker to forge tags. Poly1305 is described in
 
         :return bytes: The message authentication code as bytes.
         :raises cryptography.exceptions.AlreadyFinalized:
+
+    .. classmethod:: sign(key, data)
```

Comment:
What do you think about `generate_tag` to go alongside `verify_tag` as a name for this classmethod?

Reply:
That sounds good to me! I've updated the method name to `generate_tag`.